src/commands/nickreq/nickreq.ts

```diff
@@ -0,0 +1,78 @@
+const commando = require('discord.js-commando');
```

Comment:
Sorry, I mean that preferably you can use `import` syntax and only require if you have no other choice (e.g. the discord-buttons library)